### DISCUSSION TITLE
Custom dark theme that I use

### DISCUSSION THREAD

**Discussion by raiyansarker** [upptime/upptime#458] [Show and tell]
Hi guys,
I am using a theme that I made and tweaked a few css properties with my taste. It is inspired by Google and Github. I thought of sharing the theme, you might like it for your status page too. Here are two default themes dark themes provided by `Upptime`
![Annotation 2021-10-14 175542](https://user-images.githubusercontent.com/38852396/137312610-c6b4ad09-6a9c-4e65-820a-486d5ca50fdc.png)
![Annotation 2021-10-14 175600](https://user-images.githubusercontent.com/38852396/137312637-00e9d7f6-df2f-478e-8851-6dd6544c6ce7.png)

Here is the theme that I created,
![Annotation 2021-10-14 175647](https://user-images.githubusercontent.com/38852396/137312746-587fa5a0-8144-4545-9358-8d56890a9612.png)

You can have a demo at this [`link`](https://status.raiyansarker.com). If you want to use it, you can easily use a CDN like `jsDelivr` and here is the direct jsDelivr cdn link, [`cdn`](https://cdn.jsdelivr.net/gh/raiyansarker/status-raiyansarker@master/assets/universe.css)

Here is the repo [`link`](https://github.com/raiyansarker/status-raiyansarker/blob/master/assets/universe.css). Tweak the theme with your taste and needs.

I hope you find it helpful!

**Comment by Andre601** [comment]
Really cool design.
